{{cookiecutter.repo_name}}/package.json

```diff
@@ -444,7 +444,6 @@
       "at-rule-name-case": "lower",
       "at-rule-name-space-after": "always",
       "at-rule-no-vendor-prefix": true,
-      "stylelint-disable-reason": "always-before",
```

Comment:
I liked it, but there's nothing stopping you from continuing adding a reason above regardless of the enforcement.